### DISCUSSION TITLE
Trilium theme for the doc links

### DISCUSSION THREAD

**Discussion by ttytm** [TriliumNext/Trilium#6160] [Show and tell]
### Describe feature

It's quite a while ago that I made themes for trillium notes.

I never really pushed them or made the effort to promote them. Maybe some people like them and noone currently sees them.
I think they came out quite nice, so I thought to ask you if you want to link them in the themes part of your doc and wherever you link the other themes for trilium.

This is the link:
https://github.com/tobealive/trilium-midnight-theme
Just gave this one a code cleanup. Let me know if I'm right asking for this here and if anything else is necessary from my side.

**Comment by zadam** [comment]
Very nice! I added the link to https://github.com/zadam/trilium/wiki/Themes (it's wiki BTW, you can edit the page yourself if needed)

**Comment by AKmatiAK** [comment]
I just installed your theme today (before reading this) xD

edit: based on dark+?

**Comment by ttytm** [comment]
Hey, 

Nice to read that 😁. 

About it's base: It came from the time I was using SiYuan a lot. The main theme is also called midnight and I really liked it.
But it's very dark-plus-ish. It was probably based on that. So, yeah, describe it as you like.

I also made a short statement on that in the readme.

**Comment by ttytm** [comment]
> Very nice! I added the link to https://github.com/zadam/trilium/wiki/Themes (it's wiki BTW, you can edit the page yourself if needed)

Thanks a lot! I clean up the code for my other theme and do it that way.
